lib/hobbies.js

```diff
@@ -0,0 +1,76 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var request = require('request');
+var $ = require('cheerio');
+
+
+function hobbiesRoute() {
+  var hobbiesRouter = new express.Router();
```

Comment:
Don't need the "new" here. Express will do it for you.

(http://expressjs.com/4x/api.html#router)